pegasus/cache/i18n/zh-TW.yml

```diff
@@ -391,9 +391,9 @@
   homepage_header_donors: Code.org 感謝所有支持者
   homepage_donors_seall: 檢視全部
   homepage_donors_donate: 贊助
-  homepage_below_hero_announcement_text: We care about your privacy and are making
-    some updates to our privacy policy, effective May 25.
-  homepage_below_hero_announcement_link_text: 繼續學習
+  homepage_below_hero_announcement_text: 我們尊重您的隱私權，並且正在對我們的隱私權政策做一些更新
+    更新後的政策將於 5 月 25 日生效。
+  homepage_below_hero_announcement_link_text: 瞭解更多資訊
```

Comment:
Third instance. Volunteer translation is more like "continue studying" while professional translation is "learn more." Keeping professional translation.